.circleci/config.yml

```diff
@@ -41,21 +47,18 @@ commands:
     description: "Add changes to SVN"
     steps:
       - run:
-          command: if [[ ! -z $(svn st | grep ^\!) ]]; then svn st | grep ^! | awk '{print " --force "$2}' | xargs -0r svn rm; fi
-      - run: svn add --force .
+          command: |
+            svn stat | { grep -E '^\?' || true; } | awk '{print $2}' | xargs -r svn add
+            svn stat | { grep -E '^\!' || true; } | awk '{print $2}' | xargs -r svn rm
```

Comment:
This is the main fix. It removes files in svn that were deleted, like `svgxuse.js`.